app/exporters/manual_publishing_api_exporter.rb

```diff
@@ -1,11 +1,14 @@
 class ManualPublishingAPIExporter
+  UPDATE_TYPES = %w(minor major republish).freeze
```

Comment:
I don't think we would expect differences, no.  I'll think about what a good place to store this might be for all of them.